internal/plan/plan.go

```diff
@@ -46,6 +46,7 @@ type Plan struct {
 	Layers   []*Layer            `yaml:"-"`
 	Services map[string]*Service `yaml:"services,omitempty"`
 	Checks   map[string]*Check   `yaml:"checks,omitempty"`
+	Logging  *Logging            `yaml:"logging,omitempty"`
```

Comment:
It still sits a bit weirdly with me that the other top-level types (layers and checks) are maps of string to thing, whereas this is a struct containing a map, and other things. I wonder if we can iron that out; or maybe I'm overthinking it and it's fine.